### PR TITLE
Chart-API: Minor style fixes

### PR DESCRIPTION
**Pull request by sdobbert**
Some minor fixes for the Chart-API:
- Fixes threshold line, which is now displayed in the chart
- moves toolbox to fix overlapping title

@chrismayer @hblitza

### CODE REVIEW

**Review by chrismayer — APPROVED** [review]
Code looks good :+1:

**Comment by chrismayer** [comment]
One minor thought: Is there a default value for the threshold or will the request fail if no threshold is provided?

**Comment by sdobbert** [comment]
Thank you for the suggestion! I added a default value for the threshold (30).

File: easy-to-use-api/klips-chart-api/src/components/Chart/index.ts [hunk 1]
```diff
@@ -175,45 +175,46 @@ export class ChartAPI {
     });
 
     // check if threshold is given
-    if (this.params && this.params.threshold) {
+    if (!this.params.threshold) {
+    this.params.threshold = '30';
```
Comment (chrismayer) [inline easy-to-use-api/klips-chart-api/src/components/Chart/index.ts:179]: Wrong indention

**Pull request merged**